easypost/__init__.py

```diff
@@ -791,10 +791,11 @@ class PostageLabel(AllResource, CreateResource):
 
 class Tracker(AllResource, CreateResource):
     @classmethod
-    def create_list(cls, api_key=None, **params):
+    def create_list(cls, params={}, api_key=None):
```

Comment:
`api_key` is the first param to all functions in this library. Please adjust the ordering here.

Reply:
If I swap back to the one we had, the user has to pass a **None** in the first parameter because we are no longer using keyword arguments in the parameters.